### ISSUE TITLE
Implement WCAG keyboard inputs for AccessibleDropdown

### ISSUE THREAD

**Issue by ShrimpCryptid** [allen-cell-animated/timelapse-colorizer#513]
## Use Case

Extension off of #408 ("Align dropdowns with WCAG rules (keyboard inputs") and #428 ("refactor: Use react-select for selection dropdowns"). Implement WCAG/Aria spec in `AccessibleDropdown`, currently used for the Help menu.

## Acceptance Criteria

- `AccessibleDropdown` should open only on click (not on hover) or spacebar button input.
- Up/down arrow keys should move the focused element in the list, and clicking again with the spacebar should select it and close the menu.
- Home/End should move to the top or bottom of the dropdown list.
- Styling/visual appearance should be identical to the original dropdowns.

## Details
- Currently this is only used in Help dropdown. 
- Look into WCAG rules for menus?
- Can we use `react-select`? Another component?
